src/main/scala/edu/ie3/mobsim/MobilitySimulator.scala

```diff
@@ -275,6 +273,8 @@ final class MobilitySimulator(
   ): (Map[UUID, Int], Seq[Movement]) =
     evs.par.toSeq.flatMap(handleDepartingEv) match {
       case movements =>
+        updateElectricVehicles(movements.seq)
```

Comment:
I think that merge went wrong. `updateElectricVehicles` is now called two times for departures

Reply:
This should be fixed now.